android/src/main/java/com/reactnativeshippedsuitesdk/RNWidgetView.kt

```diff
@@ -68,6 +71,18 @@ class RNWidgetView(context: ThemedReactContext) : FrameLayout(context) {
         config.currency = currency
       }
     }
+
+    if (configuration.hasKey("appearance")) {
+      val appearance = configuration.getInt("appearance")
+      if (appearance != null) {
+        when (appearance) {
+          0 -> config.appearance = ShippedSuiteAppearance.AUTO
+          1 -> config.appearance = ShippedSuiteAppearance.LIGHT
+          2 -> config.appearance = ShippedSuiteAppearance.DARK
+        }
+      }
+    }
```

Comment:
@hoyelo but why do you need this int -> enum code?

Reply:
The declaration of this enum in JavaScript is like this:
```
export enum ShippedSuiteAppearance {
  Auto,
  Light,
  Dark,
}
```
but the enum variable obtained in Android is an integer, so it has to be converted to the enum type defined in Android, which looks like this:
```
enum class ShippedSuiteAppearance(val value: String) {
    LIGHT("light"), DARK("dark"), AUTO("auto");
}
```

Reply:
hmmm not fully understanding it... but ok